docs/source/user-guide/configuration/use-condarc.rst

```diff
@@ -161,6 +182,12 @@ This is especially useful for air gap and enterprise installations:
     - http://some.custom/channel
     - file:///some/local/directory
 
+As of conda 3.7, the ``free`` channel is no longer part of the
+``defaults`` channel. If you want to add ``free`` back into your
```

Comment:
So this section is explaining how to redefine "defaults" in your .condarc file from the standard defaults that come with conda.    The free channel is part of the standard "defaults" - but if someone sets the default channels in .condarc - then only the ones they list are searched by default.  In other words, if someone adds this section into their .condarc - and the free or main channel isn't there, then it isn't in "defaults" - regardless of which version of conda.

Reply:
Also good clarification.  We should add a reference to the ``default_channels`` config parameter docs.

Reply:
It seems like it doesn't really make sense to bring up the ``free`` channel in this section then?

@msarahan - what docs should I refer to? https://docs.conda.io/projects/conda/en/latest/configuration.html ?

Reply:
This one was what I had in mind:

https://docs.conda.io/projects/conda/en/latest/user-guide/configuration/use-condarc.html?highlight=default_channels#default-channels-default-channels

Reply:
That link is for the same section we're editing in. I removed the reference to the ``free`` channel entirely for this part.